src/ngraph/deprecated.hpp

```diff
@@ -31,6 +31,8 @@
 //
 #ifdef NGRAPH_DEPRECATED_ENABLE
 #define NGRAPH_DEPRECATED(msg) __attribute__((deprecated((msg))))
+#define NGRAPH_DEPRECATED_DOC /// \deprecated
```

Comment:
I think these crept in from another PR.

Reply:
Actually no, look down at runtime/tensor.hpp and you will see where they are used. It is needed because I added a doc \deprecated tag and that does not work if you don't use these changes.